ui/src/generated/defaults.js

```diff
@@ -8,5 +8,13 @@ export default {
   ],
   "AMM_NAME": "amm",
   "BRIDGE_URL": "http://127.0.0.1:8000",
-  "CONTRACT_NAME": "VaultFactory"
+  "CONTRACT_NAME": "VaultFactory",
```

Comment:
Please edit `ui/use-on-chain-config.js` instead, and run it to generate this file.

Reply:
Updated, but leaving this checked in as discussed